rotkehlchen/tests/fixtures/rotkehlchen.py

```diff
@@ -102,6 +103,13 @@ def fixture_perform_nodes_insertion():
     return True
 
 
+@pytest.fixture(name='premium_remote_data')
```

Comment:
This fixture should not be here. It's only used in the `test_premium.py` folder so it should go there.